server/platform/services/cache/provider.go

```diff
@@ -100,6 +101,17 @@ func NewRedisProvider(opts *RedisOptions) (Provider, error) {
 		// meant to be used at very high scales. The docs suggest 20us,
 		// but going as high as 250us doesn't make any material difference.
 		MaxFlushDelay: 250 * time.Microsecond,
+		RetryDelay: func(attempts int, _ rueidis.Completed, _ error) time.Duration {
+			// We only retry once.
+			if attempts > 1 {
+				return -1
+			}
```

Comment:
Does the `attempts` counter increase per-call or globally (asking since the client is shared)?

Reply:
It is per call.